common/src/main/java/org/conscrypt/OpenSSLSocketFactoryImpl.java

```diff
@@ -16,20 +16,19 @@
 
 package org.conscrypt;
 
-import static org.conscrypt.Platform.getFileDescriptor;
-
 import java.io.IOException;
 import java.net.InetAddress;
 import java.net.Socket;
 import java.net.UnknownHostException;
 import java.security.KeyManagementException;
 
 public class OpenSSLSocketFactoryImpl extends javax.net.ssl.SSLSocketFactory {
-    private static boolean alwaysUseSocketEngine =
-            Boolean.parseBoolean(System.getProperty("org.conscrypt.alwaysUseEngineSocket"));
```

Comment:
Minimally, I'll just need to update a whitelist to access the new name.  Thanks for calling this out!